app/controllers/users_controller.rb

```diff
@@ -0,0 +1,4 @@
+class UsersController < ApplicationController
+	def show
+	end
+end
```

Comment:
Please fix the indentation issues.
@Junaid-ahmed58 I already told you to use 2 spaces for indentations